includes/polling/os/eltek.inc.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * valere.inc.php
+ *
+ * -Description-
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2017 Tony Murray
+ * @author     Tony Murray <murraytony@gmail.com>
+ */
+
+$hardware = snmp_get($device, 'powerSystem.powerSystemModel.0', '-Ovqa', 'SP2-MIB') . ' ' . snmp_get($device, 'powerSystem.powerSystemNominalVoltage.0', '-Ovqa', 'SP2-MIB'). 'v Nominal voltage with ' . snmp_get($device, 'rectifiers.rectifiersNumberOfRectifiers.0', '-Ovqa', 'SP2-MIB') . ' rectifiers of type ' . snmp_get($device, 'rectifiers.rectifierTable.rectifierEntry.rectifierType.1', '-Ovqa', 'SP2-MIB');
```

Comment:
Please convert all snmp_gets into a single snmp_get_multi_oid(). However I'm not sure that hardware value is right. This is used to distinguish the hardware at a basic level (used in searches and the dropdown on device pages) so something this long is ott imho.